WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -187,20 +190,26 @@ extension ReaderTabViewModel {
         settingsPresenter.present(on: from, animated: true, completion: nil)
     }
 
-    func presentFilter(from: UIView, completion: @escaping (ReaderAbstractTopic?) -> Void) {
-        filterTapped?(from, { [weak self] topic in
-            if let topic = topic {
-                self?.setFilterContent(topic: topic)
+    func didTapStreamFilterButton(with filter: FilterProvider) {
+        // TODO: @dvdchr Figure out the source rect.
```

Comment:
I'll address this in the next PR. Gonna have to get a `CGRect` of the SwiftUI button (probably through GeometryReader) and then create a transparent UIView from the view controller side and place it on top of the button so that the bottom sheet will have something to anchor to when displayed on iPad...

Reply:
Plus, removing the transparent UIView once we're done with the sheet.